charts/govuk-apps-conf/templates/router-configmap.yaml

```diff
@@ -22,6 +22,12 @@ data:
       include       /etc/nginx/mime.types;
       default_type  application/octet-stream;
 
+      client_body_temp_path /tmp/client_temp;
+      proxy_temp_path       /tmp/proxy_temp_path;
+      fastcgi_temp_path     /tmp/fastcgi_temp;
```

Comment:
optional: It shouldn't use these last three anyway (at least in theory) but it doesn't hurt to set them. Up to you if you wanna leave 'em in.

Reply:
thanks for pointing that out. I will remove them and see if its happy - as no need to have unneeded config.

Reply:
will remove
 ```
 fastcgi_temp_path     /tmp/fastcgi_temp;
       uwsgi_temp_path       /tmp/uwsgi_temp;
       scgi_temp_path        /tmp/scgi_temp;
```